<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddImageServer</class>
 <widget class="QDialog" name="AddImageServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLineEdit" name="pixelLengthLineEdit"/>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="ImageFileLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="imageKind">
     <property name="text">
      <string>ImageKinds</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>ImageFile</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ph">
     <property name="text">
      <string>PhotoDate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="imageNameLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ImageName</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>PatientName</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="photoDateLineEdit"/>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="fileDialogButton">
     <property name="text">
      <string>file dialog</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="pixelLength">
     <property name="text">
      <string>PixelLength</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="patientNameLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="imageKindsLineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
